mgmt-fixes/deploy/helm/mgmt-fixes/templates/ds-kubelet-parameters.yaml

```
@@ -0,0 +1,75 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  labels:
+    component: kubelet-parameters
+  name: kubelet-parameters
```

```suggestion
  name: set-kubelet-parameters-for-scale
```
Or something else more descriptive maybe?

---

More descriptive more better, so +1 from me.